README.md

```diff
@@ -137,6 +137,38 @@ echo '{
 http://localhost:8116
 ```
 
+get cells by lock script and filter by empty type script
```

Comment:
Please clarify the definition of empty type script (`packed::ScriptBuilder::default()`)